packages/components/components/elvis-datepicker/package.json

```diff
@@ -16,10 +16,10 @@
     "dist"
   ],
   "dependencies": {
-    "@elvia/elvis-assets-icons": "^3.10.0",
-    "@elvia/elvis-colors": "^4.4.1",
+    "@elvia/elvis-assets-icons": "^3.11.1",
+    "@elvia/elvis-colors": "^4.4.2",
     "@elvia/elvis-component-wrapper": "^4.2.0",
-    "@elvia/elvis-toolbox": "^12.0.3",
+    "@elvia/elvis-toolbox": "^12.0.5",
     "@elvia/elvis-typography": "^2.7.1",
```

Comment:
Could probably update the dependency on typography here too, we should get all the components up to the newest major!